src/bundle/Resources/views/themes/standard/content_view/embed-inline.html.twig

```diff
@@ -0,0 +1,7 @@
+{% set content_name=ez_content_name(content) %}
```

Comment:
Can't we use just `{{ content.name }}`?

Reply:
Great catch, SA-aware repo is indeed injected when rendering it. Fixed via fixup 5b28f0f. I also changed template name to `embed_inline.html.twig` so it follows existing naming convention for templates.